### DISCUSSION TITLE
Is there a timeline for how long GitHub Sponsors will be beta? Kindly check my extrenuating details!

### DISCUSSION THREAD

**Discussion by staccDOTsol** [community/community#22967] [New to GitHub]
<p>Hi all, </p>
<p>I joined the GitHub sponsors waitlist a little while ago. Since I generally write stuff that arbitrages or otherwise makes money, people are happy to give me money to support my work - in tiers, which works with GitHub’s offering - and doubling my first $5k sounds like a good plan to start!</p>
<p>Extrenuating details, you wonder? TL;DR I’ve been a 0-rate services exporter for a long time, as far as federal state and provincial entities care. If I did anything or sold anything to anyone in Canada I’d need to apply GST, and if I were the kind of alien that visits US shores it’d complicate things… I’m a Canadian who wholly owns a Canadian-Controlled Private Corporation. I have spent no time physically within the US borders in more than 10 years. I have an EIN from the IRS for taxation.</p>
<p>Last few tax guys I talked to I think said a half-filld w8-BEN is all I need to be compliant.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/caeb800ff0c3393a8facc4c18382d6581a36adf2.png" data-download-href="/uploads/short-url/sX70mXYhkO27hi0yztFURPm3v1g.png?dl=1" title="Selection_110.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/22843601/181074073-6c36f24d-90b0-43e9-a19b-6d145539548c.png" alt="Selection_110.png" data-base62-sha1="sX70mXYhkO27hi0yztFURPm3v1g" width="690" height="233" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/caeb800ff0c3393a8facc4c18382d6581a36adf2_2_690x233.png, https://aws1.discourse-cdn.com/github/original/1X/caeb800ff0c3393a8facc4c18382d6581a36adf2.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/caeb800ff0c3393a8facc4c18382d6581a36adf2.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/caeb800ff0c3393a8facc4c18382d6581a36adf2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Selection_110.png</span><span class="informations">889×301 38 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I don’t have ‘Sponsors dashboard’ that I can see. <a href="https://help.github.com/en/articles/becoming-a-sponsored-developer" rel="nofollow noopener">https://help.github.com/en/articles/becoming-a-sponsored-developer</a> is this doc out of date?</p>
<p>I have a small and growing community of sometimes or potentially-in-the-future contributing funds so I can focus on making bots for them, or looking to contribute their ideas. I believe at least a handful of people are nice enough to have done some of / all of this:</p>
<p><img src="https://user-images.githubusercontent.com/22843601/181074077-e86a1fcc-b700-4f80-a6e8-bc4f3ffa5f8b.png" alt="Selection_111.png" data-base62-sha1="lUIh2me1gueqwruvOGrstd4qZ8g" width="439" height="272"></p>
<p>Thanks for this awesome feature and thank the community for any insight you may have!</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/dunncreativess">@dunncreativess</a>,</p>
<p>Thank you for being here, Unfortunately, I can’t give you an ETA for when it will reach you, but we’re rolling it out as fast as we can! Thanks for your patience</p>

**Comment by devonzuegel** [comment, marked as answer]
<p>Good news: GitHub Sponsors is now out of beta to developers with bank accounts in <a href="https://help.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/becoming-a-sponsored-developer#submitting-your-bank-and-tax-information" rel="nofollow noopener">30 countries</a> and growing! You can learn more in <a href="http://memex.link/yyGcHtGiN/github.blog/2019-11-04-github-sponsors-is-now-out-of-beta-in-30-countries" rel="nofollow noopener">the announcement blog post</a>.</p>
<p>If you don’t have a bank account in one of the 30 countries where GitHub Sponsors is generally available, you can still sign up on the waitlist to join the beta in your country. In the coming months, we’ll continue to roll out general availability to countries where GitHub does business, incorporate feedback, and improve the tools you need to connect with the community you depend on and collaborate with.</p>
